src/test/java/com/google/javascript/gents/multiTests/converts_ts_module_require/converts_ts_module_require.js

```diff
@@ -0,0 +1,3 @@
+const {foo} = goog.require("google3.already.ts.module");
+const wholeModule = goog.require("google3.already.ts.module");
```

Comment:
should there be a test for 

// this one shouldn't match
const wholeModule = goog.require("google3.already.tsomething.else");